Source/ResponseSerialization.swift

```diff
@@ -515,12 +515,23 @@ extension DownloadRequest {
 }
 
 // MARK: - Empty
+/// A protocol for a type representing an empty response. Use `T.emptyValue` to get an instance.
+public protocol EmptyResponse {
+    static func emptyResponseValue() -> Self
```

Comment:
I like the transition to a function here, but this seems rather verbose. Perhaps just `emptyValue()`?

Reply:
addressed in 4664bb578a70c3f8a1d3f74082b352d5eea8fd06